libs/features/georoutingv2/georoutingv2.css

```diff
@@ -140,19 +136,6 @@
   padding: initial;
 }
 
-@media (min-width: 480px) {
-  .dialog-modal.locale-modal-v2 {
-    background-image: url('/libs/features/georoutingv2/img/GeoModal_BG_Map_Tablet.png');
```

Comment:
we need to remove the image from assets

Reply:
@narcis-radu I have removed this image from repo